### PR TITLE
feat: Add cluster use command for created cluster

### PR DESCRIPTION
**Pull request by bbejeck**
Sets the newly created cluster as the one to use for subsequent commands

### CODE REVIEW

File: confluent-cloud_kickstart/confluent-cloud_kickstart.py [hunk 1]
```diff
@@ -95,6 +95,9 @@ def write_to_file(file_name, text, json_fmt=True):
 print("Enabling the API key for the Kafka cluster")
 cli(["confluent", "api-key", "use", creds_json['api_key'], "--resource", cluster_json['id']], debug, fmt_json=False)
 
+print("Setting created cluster for use in subsequent commands")
+cli(["confluent", "kafka", "cluster", "use", cluster_json['id']], debug, fmt_json=False)
+
 print("Generating client configuration")
 client_config = cli(["confluent", "kafka", "client-config", "create", args.client,
                      "--cluster", cluster_json['id'],
```
Comment (brianstrauch) [inline confluent-cloud_kickstart/confluent-cloud_kickstart.py:103]: The previous command would allow us to remove the following, right?
```suggestion
```

Comment (bbejeck) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py:103]: yep - good catch!

Comment (bbejeck) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py]: I validated this works

File: confluent-cloud_kickstart/confluent-cloud_kickstart.py [hunk 2]
```diff
@@ -95,9 +95,11 @@ def write_to_file(file_name, text, json_fmt=True):
 print("Enabling the API key for the Kafka cluster")
 cli(["confluent", "api-key", "use", creds_json['api_key'], "--resource", cluster_json['id']], debug, fmt_json=False)
```
Comment (brianstrauch) [inline confluent-cloud_kickstart/confluent-cloud_kickstart.py:96]: This flag is no longer required here and can be removed!
```suggestion
cli(["confluent", "api-key", "use", creds_json['api_key']], debug, fmt_json=False)
```

**Review by brianstrauch — APPROVED** [review]

File: confluent-cloud_kickstart/confluent-cloud_kickstart.py (hunk 2 above)
Comment (bbejeck) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py:96]: whoops! I was too fast on the push, I'll do a follow-up PR now

Comment (bbejeck) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py:96]: Hmm, I'm not sure, now that I think about it, you need to specify which resource you're generating keys for.
I'll test it out

Comment (bbejeck) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py:96]: Doing that results in an error
```
Error: required flag(s) "resource" not set
```

Comment (brianstrauch) [reply confluent-cloud_kickstart/confluent-cloud_kickstart.py:96]: `--resource` is required for creating API keys, but no longer required for using them!

**Pull request merged**